tests/models/test_modeling_value_head.py

```diff
@@ -384,6 +409,35 @@ def test_push_to_hub(self):
                     f"Parameter {name} is not the same after push_to_hub and from_pretrained",
                 )
 
+    def test_transformers_kwargs(self):
```

Comment:
Is this test duplicate? the only difference seems to be the FSMT check

Reply:
Yeah it's just that for seq2seq models `fmst` does not have any `lm_head` and as I suppose that this model is not very much used (by looking at the number of downloads) I decided to skip it.

Reply:
ah ok, can you also name it ? then we can merge
```suggestion
    def test_transformers_bf16_kwargs(self):
```

Reply:
Yes sure!